src/reference/asciidoc/file.adoc

```diff
@@ -448,6 +448,18 @@ If the tail command fails (on some platforms, a missing file causes the `tail` t
 
 [source,xml]
 ----
+<int-file:tail-inbound-channel-adapter id="native"
+	channel="input"
+	enable-status-reader="false"
+	task-executor="exec"
+	file="/tmp/foo"/>
+----
+
+By default native adapter capture from standard output and send them as messages and from standard error to raise events.
+You can discard the standard error events by setting enable-status-reader to false.
```

Comment:
Need to wrap attribute to the code formatting, but that is minor. Will fix on merge.
LGTM

Also need `TailAdapterSpec` modification for Java DSL.
Plus I don't see any sample in the doc.
All this work is for another pull request.

Feel free to consider that part as a subsequent contribution as well, @shahbour 😄